impl/src/bazel.rs

```diff
@@ -23,6 +23,35 @@ use crate::{
   util::RazeError,
 };
 
+use std::{
+  env,
+  path::{Path, PathBuf},
+};
+
+/** Returns a path to a Bazel workspace root if one could be found, otherwise None */
+pub fn find_workspace_root(current_dir: Option<&Path>) -> Option<PathBuf> {
+  let default = env::current_dir().unwrap();
+  let cwd: &Path = current_dir.unwrap_or(default.as_path());
+
+  let workspace_files = [
+    cwd.clone().join("WORKSPACE.bazel"),
+    cwd.clone().join("WORKSPACE"),
```

Comment:
Is `clone()` needed here?  It looks [`Path::join`](https://doc.rust-lang.org/std/path/struct.Path.html#method.join) returns an owned PathBuf.

Reply:
It's not, thanks, I removed the use of it here.